documentation/blog/2025-04-17-goose-goes-to-NY/index.mdx

```diff
@@ -0,0 +1,123 @@
+
+- **[Erin Mikail](https://www.linkedin.com/in/erinmikail/)** from Galileo ran a Goose-themed game that got people laughing *and* learning about the importance of agent evaluation.  
+- **[I, Ebony Louis](https://www.linkedin.com/in/ebonylouis/)** gave a walkthrough of Goose and MCP, showing how quick it is to get started with any MCP server.  
+- **[Alex Hancock](https://www.linkedin.com/in/alexjhancock/)** went deep on MCP, and how it powers secure, extendable agent workflows.
+
+Then came the best part — hack time. **27 people got Goose up and running live** during the event, which was amazing to see. It’s always fun watching that moment when it clicks for someone, ideas start flowing, people get inspired, and they walk away ready to actually *use* Goose. 
+
+## What People Said
+
+<ImageCarousel id="test-carousel" images={[swag, swag1, swag2, swag3]} />
+
+
+> “It took me less than 5 mins to get myself set up with a sample project and running the Goose CLI in GitHub Codespaces! Overall it felt great to be back in a physical venue for a hands-on event with an engaged community and a timely topic for discussion.”  
+> — *Nitya Narasimhan, AI @ Microsoft*
+
+> “The live demos, open discussions, and practical exercises made it an exceptional learning experience. Feeling energized and inspired to explore how Goose and MCP can be leveraged in future data engineering projects!”  
+> — *Manikumarreddy Gajjela, Full Stack Developer*
+
+> “Everyone seems bullish on Model Context Protocol (MCP) these days… Goose is an open-source, on-machine AI agent built to automate your tasks. The coolest part? You can even plug in your own hosted Ollama LLM with Goose!”  
+> — *Rijul Dahiya, NYU Courant*
+
+> “Great event – great mix of introductory and technical material as well as fun, and I liked that it provided a bunch of time at the beginning and end that was unstructured.”  
+> — *Matthew Hill*
+
+## Moments We Loved ❤️
+
+- 🍕 Pizza, music, and people catching up before things even kicked off  
+- 🎮 The Goose game- yes, it got competitive and oddly dark lol
+- 🧳 Folks traveled from Philly and DC just to attend  
+- 📸 Having a photographer meant we could actually *enjoy* the night  
+- 🧠 A bunch of thoughtful “I’m not a dev but…” questions that we *loved* hearing  
+
+<ImageCarousel id="test-carousel" images={[speaker1,speaker, fun, fun1, focus2]} />
+
+At the end, people were asking when the next one is and yes, we may have let it slip early…
+
+## Up Next: Goose Flies South 🛫🐣
+
+Our next Goose Meetup is already locked in!
+
+👉 **Atlanta – Wednesday, April 30**  
+🕕 **6:00 PM – 8:30 PM ET**  
+📍 [RSVP here](https://lu.ma/x9ccqruq)
+
+## Things We’re Thinking About for Next Time
+
+Every event teaches us something. Here’s what we’re carrying forward:
+
+- **Let everyone off the waitlist 2 days before.** RSVP drop-offs are real, and this keeps the energy high.
```

Comment:
wait i feel like we should leave this part out and keep that as info for our team only. feel free to let me know why you're including this.

Reply:
okay i like the what we learned..but i dont wanna tell them about the waitlist thing yet LOLLL.